using-packages.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: page
+title: Using Packages
+permalink: /using-packages/
+---
+
+Flutter supports using shared packages contributed by other developers to the
+Flutter and Dart ecosystems. This allows you to quickly build your app without
+having to develop everything from scratch.
+
+Existing packages enable many use cases, for example, file path manipulation
+([`path`](https://pub.dartlang.org/packages/path)), custom navigation/route
```

Comment:
nit: is path really relevant to Flutter developers? What about a JSON library? or, better yet: http package